### PR TITLE
Improved the OVAL lab.

### PR DESCRIPTION
**Pull request by matejak**
* Improved lab's clarity.
* Enabled the RHEL7 podman image for the test suite.

Fixes: #131

### CODE REVIEW

File: 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc [hunk 1]
```diff
@@ -173,53 +190,101 @@ The test for the `/etc/profile` criterion and it's dependencies look like this:
...
+The state is a specification that the object (i.e. the matched substring) should be an integer that equals to the value of a variable `var_accounts_tmout`.
+
 There are two regular expression that check for `TMOUT=...` in the `shared.xml` file, one for the `profile` test, and one for the `profile.d/*.sh` test.
-However, they are the same, and the alternative form of the assignment `export TMOUT=...` isn't handled in either of them.
+As there are two types of locations that need to be examined (the single `/etc/profile` file and `*.sh` files in the `/etc/profile.d` directory), there have to be two objects.
+The `object_etc_profile_tmout` and `object_etc_profiled_tmout` objects have different file / path specifications, but the regular expression is the same.
+The alternative form of the assignment `export TMOUT=...` isn't handled in either of them.
 
 Moreover, there is the `equals` operation used to perform the match.
 As stated in the previous section, this looks wrong, as shorter timeouts are more secure, and therefore should be allowed.
```
[the remark this answers is not in the document]
Comment (matejak) [reply 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc]: I believe that `this looks wrong` is correct - it is about the comparison operation, not about the duplicate occurrence of the string.

File: 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc [hunk 2]
```diff
@@ -173,53 +190,101 @@ The test for the `/etc/profile` criterion and it's dependencies look like this:
...
+We see that the `test_etc_profile_tmout` test references the `object_etc_profile_tmout` object and the `state_etc_profile_tmout` state.
+The object definition associates a filename with a regular expression.
+The filename is checked for the regular expression, and if there is a match, contents of the regular expression group become the object.
+The `instance` element that equals 1 tells us that it is the first match of the regular expression what defines the object.
+
+The state is a specification that the object (i.e. the matched substring) should be an integer that equals to the value of a variable `var_accounts_tmout`.
+
 There are two regular expression that check for `TMOUT=...` in the `shared.xml` file, one for the `profile` test, and one for the `profile.d/*.sh` test.
-However, they are the same, and the alternative form of the assignment `export TMOUT=...` isn't handled in either of them.
+As there are two types of locations that need to be examined (the single `/etc/profile` file and `*.sh` files in the `/etc/profile.d` directory), there have to be two objects.
```
[the remark this answers is not in the document]
Comment (matejak) [reply 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc]: This is interesting - here I had to escape the asterisk using `\*`, but escaping that one elsewhere in inline code segments would result in literal `\*` appearing in the HTML output.

File: 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc [hunk 3]
```diff
@@ -158,6 +158,14 @@ We have the leading `definition` element:
...
 Here is a close-up of those critera:
 
+----
+    ...
+    <criteria operator="OR">
+      <criterion comment="TMOUT value in /etc/profile >= var_accounts_tmout"        test_ref="test_etc_profile_tmout" />
+      <criterion comment="TMOUT value in /etc/profile.d/*.sh >= var_accounts_tmout" test_ref="test_etc_profiled_tmout" />
+    </criteria>
+  </definition>
+  ...
```
Comment (yuumasato) [inline 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc]: @matejak The snippet is duplicated. And from this section many paragraphs are in verbatim mode.

Comment (matejak) [reply 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc]: Thanks for noticing, I have fixed that.

File: 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (yuumasato) [reply 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc]: Yes, I mean the comparison operation, but I don't have a strong opinion here.

**Comment by yuumasato** [comment]
The changes looked good to me. :)

**Pull request merged**